source/classes.tex

```diff
@@ -4158,31 +4150,17 @@
 \end{codeblock}
 \end{example}
 
-\pnum
-\begin{note}
-An abstract class type cannot be used
-as a parameter or return type of
-a function being defined\iref{dcl.fct} or called\iref{expr.call},
-except as specified in \ref{dcl.type.simple}.
-Further, an abstract class type cannot be used as
-the type of an explicit type conversion~(\ref{expr.static.cast},
-\ref{expr.reinterpret.cast}, \ref{expr.const.cast}),
-because the resulting prvalue would be of abstract class type\iref{basic.lval}.
-However, pointers and references to abstract class types
-can appear in such contexts.
-\end{note}
-
 \pnum
 \indextext{function!virtual!pure}%
-A class is abstract if it contains or inherits at least one pure virtual
-function for which the final overrider is pure virtual.
+A class is an \defnadj{abstract}{class} if it has at least one member that is
```

Comment:
Semantic line breaks, please: the stuff after (and including) the "if" should be on a single separate line.